include/ddc/kernels/splines/null_extrapolation_rule.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+namespace ddc {
+
+struct NullExtrapolationRule
+{
+    explicit NullExtrapolationRule() {}
+
+    template <class CoordType, class ChunkSpan>
+    KOKKOS_FUNCTION double operator()(CoordType, ChunkSpan) const
```

Comment:
We need to add constraints about `CoordType` and `ChunkSpan`

Reply:
What constraints ? We do not even have access to a DimI here because it would be useless.

Reply:
As we discussed on slack, the checks should be handled in `SplineEvaluator`